buildconfig/Jenkins/buildscript

```diff
@@ -58,7 +58,9 @@ terminate_xvfb_sessions
 # System discovery
 ###############################################################################
 USE_CORE_DUMPS=true
-if [[ ${NODE_LABELS} == *rhel7* ]] || [[ ${NODE_LABELS} == *centos7* ]] || [[ ${NODE_LABELS} == *scilin7* ]]; then
+if [[ ${JOB_NAME} == *master_conda* ]]; then
+    ON_CONDA=true
+    elif [[ ${NODE_LABELS} == *rhel7* ]] || [[ ${NODE_LABELS} == *centos7* ]] || [[ ${NODE_LABELS} == *scilin7* ]]; then
```

Comment:
I know you haven't touched the other `elif` clauses but could you unindent them for readability of this block.